pimd/pim_zebra.c

```diff
@@ -580,6 +580,23 @@ static int pim_zebra_vxlan_sg_proc(ZAPI_CALLBACK_ARGS)
 	return 0;
 }
 
+static void pim_zebra_vxlan_replay(void)
+{
+	struct stream *s = NULL;
+
+	/* Check socket. */
+	if (!zclient || zclient->sock < 0)
+		return;
+
+	s = zclient->obuf;
+	stream_reset(s);
+
+	zclient_create_header(s, ZEBRA_VXLAN_SG_REPLAY, VRF_DEFAULT);
```

Comment:
Shall we pass vrf_id from the caller of this function and use it instead of hardcoding at the lower level function?

Reply:
This replay request is not really per-vrf thing; its global (independent of the underlay VRF id).